app/models/pawn.rb

```diff
@@ -18,17 +18,17 @@ def double_advance?(to_row, to_col)
   end
 
   def capture_move?(to_row, to_col)
-    to_row == row + forward_one && (to_col - col).abs == 1 && enemy_occupied?(to_row, to_col)
+    to_row == row + forward_one && (to_col - col).abs == 1 && enemy_at(to_row, to_col)
   end
 
   def en_passant?(to_row, to_col)
     return false unless to_row == row + forward_one && (to_col - col).abs == 1
-    enemy = enemy_occupied?(to_row + back_one, to_col)
+    enemy = enemy_at(to_row + back_one, to_col)
     enemy && enemy.class == Pawn && enemy.en_passant_vuln?
```

Comment:
Just an interesting note - apparently `protected` methods like `en_passant_vuln?` can be called from other instances of the same class.  TIL